tools/build_defs/shell_toolchain/toolchains/impl/default_commands.bzl

```diff
@@ -72,8 +72,11 @@ elif [[ -L "$1" ]]; then
   local actual=$(readlink "$1")
   ##symlink_contents_to_dir## "$actual" "$target"
 elif [[ -d "$1" ]]; then
-  local children=$(find -H "$1" -maxdepth 1 -mindepth 1)
-  for child in $children; do
+  SAVEIFS=$IFS
+  IFS=$'\n'
+  local children=($(find -H "$1" -maxdepth 1 -mindepth 1))
+  IFS=$SAVEIFS
```

Comment:
First time I see this trick… I looked into it a bit and found [this article describing a similar approach](https://www.cyberciti.biz/tips/handling-filenames-with-spaces-in-bash.html).

Overall 👍.